Cargo.toml

```diff
@@ -168,6 +168,8 @@ rustdoc.all = "warn"
 # suggestions which we fixed. The others didn't have any findings, so we can
 # assume they don't have that many false positives. Let's enable them to
 # prevent future problems.
+as_ptr_cast_mut = "allow"
```

Comment:
I think that if we `allow` this, we don't need to move it here no? Only warnings are in this list imo

Reply:
You're right! Reverted this change to add it and fix lints in separate PR.